app/core/src/main/java/com/fsck/k9/controller/MessagingController.java

```diff
@@ -2010,6 +2061,9 @@ private void moveOrCopyMessageSynchronous(final Account account, final String sr
                     for (Entry<String, Message> entry : origUidMap.entrySet()) {
                         String origUid = entry.getKey();
                         Message message = entry.getValue();
+                        if (operation == MoveOrCopyFlavour.MOVE_AND_MARK_READ) {
+                            setFlag(account, localDestFolder.getServerId(), new ArrayList<>(messages), Flag.SEEN, true);
```

Comment:
Hey, I would prefer if deleted mails would also marked as read on the server, not only locally. Otherwise, if I synchronize my trash-folder I would get unread mails again.

Reply:
> Hey, I would prefer if deleted mails would also marked as read on the server, not only locally.

They are marked as read on the server, too. Just not in this line of code :)